common/core/web/tests/resources/json/engine_tests/ghp_enter.json

```diff
@@ -0,0 +1,41 @@
+{
+  "specVersion": "14.0",
+  "keyboard": {
+    "id": "galaxie_hebrew_positional",
+    "name": "Galaxie Hebrew (Positional)",
+    "filename": "resources/keyboards/galaxie_hebrew_positional-2.2.js",
+    "languages": [
+      {
+        "id": "he",
+        "name": "Hebrew",
+        "region": "Asia"
+      }
+    ]
+  },
+  "inputTestSets": [
+    {
+      "constraint": {
+        "target": "desktop",
+        "validOSList": null,
+        "validBrowsers": null
+      },
+      "testSet": [
+        {
+          "inputs": [
+            {
+              "type": "osk",
+              "layer": "default",
+              "keyName": "K_M"
+            },
+            {
+              "type": "osk",
+              "layer": "default",
+              "keyName": "K_ENTER"
+            }
+          ],
+          "output": "ם\n"
```

Comment:
So this doesn't look like it tests the final-sigma case; is that what you are expecting here?

Reply:
That _is_ the exact test you're saying you want.

```
+ "m" > U+05DE
U+05DE + [K_TAB] > U+05DD use(outkey)
```

Also works with `[K_ENTER]`.

Refer to https://github.com/keymanapp/keyman/pull/4350#issuecomment-768100099.

The default "m" output:  `מ`
The second rule transforms that into a `ם` as seen in the specified `"output"` used by the test's assert clause.

`מ` vs `ם`... it can be rather subtle without zooming in.

Reply:
Ah, yes, I misread this entirely ... was expecting Greek not Hebrew and then it just didn't click. Looks good.